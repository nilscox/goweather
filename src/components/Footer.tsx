/** @jsx jsx */
import { css, jsx } from '@emotion/core'

/**
 * Displays a footer with some very useful links.
 */
const Footer = () => (
  <footer css={footerStyle} className="text-muted text-center py-2">
    Made with â™¥ by <a className="text-muted" href="https://github.com/nilscox">nilscox</a> for <a className="text-muted" href="https://gojob.com">gojob</a>.
  </footer>
);

const footerStyle = css`
  font-size: 10px;
`;

export default Footer;
