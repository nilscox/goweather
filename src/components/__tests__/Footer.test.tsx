import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom';

import Footer from '../Footer';

test('Footer', () => {
  const tree = renderer
    .create(<Router><Footer /></Router>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
