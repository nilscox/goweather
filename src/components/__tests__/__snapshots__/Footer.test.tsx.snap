// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer 1`] = `
<footer
  className="text-muted text-center py-2 css-1l7bpnt"
>
  Made with â™¥ by 
  <a
    className="text-muted"
    href="https://github.com/nilscox"
  >
    nilscox
  </a>
   for 
  <a
    className="text-muted"
    href="https://gojob.com"
  >
    gojob
  </a>
  .
</footer>
`;
